src/npcmove.cpp

```diff
@@ -219,24 +219,11 @@ const std::vector<bionic_id> weapon_cbms = { {
 
 const int avoidance_vehicles_radius = 5;
 
-bool good_for_pickup( const item &it, npc &who )
+bool good_for_pickup( const item &it, npc &I, const tripoint &there )
```

Comment:
Maybe keep `who` as variable name since that's how it's called in other functions?

Reply:
I came, I saw, I got code style'd
(Yes I agree code style is more important than funny commit names, it has been done.)